torch/optim/radam.py

```diff
@@ -379,16 +396,29 @@ def _multi_tensor_radam(
         if grouped_state_steps[0].is_cpu:
             torch._foreach_add_(grouped_state_steps, torch.tensor(1.0, device='cpu'), alpha=1.0)
         else:
-            torch._foreach_add_(grouped_state_steps, 1)
+            torch._foreach_add_(grouped_state_steps, 1.0)
```

Comment:
wait why is this necessary

Reply:
not necessary, I just did it since we do it in most other places

Reply:
if you care, I can remove it.

Reply:
Where else do you see 1.0?

I'd prefer this not be here if there's no reason

Reply:
There is an instance with ASGD as well, but I'll just remove idc